Fix: Use the 'short' instead of 'tiny' date format for NewGRF string code 83.

diff -r b0731a6e08ab src/newgrf_text.cpp
--- a/src/newgrf_text.cpp	Sat Aug 20 14:18:14 2011 +0200
+++ b/src/newgrf_text.cpp	Sat Aug 20 15:10:43 2011 +0200
@@ -1067,7 +1067,7 @@
 			return SCC_DATE_LONG;
 
 		case SCC_NEWGRF_PRINT_MONTH_YEAR:
-			return SCC_DATE_TINY;
+			return SCC_DATE_SHORT;
 
 		case SCC_NEWGRF_PRINT_WORD_SPEED:
 			return SCC_VELOCITY;
